src/index.js

```diff
@@ -1,25 +1,28 @@
 // eslint-disable-next-line no-unused-vars
 import _ from 'lodash';
 import './style.css';
-import Leaderboard from './modules/leaders.js';
+import { postScore } from './modules/leaders.js';
 import UI from './modules/UI.js';
-import LeaderboardStore from './modules/leadersData.js';
 
 const form = document.getElementById('form');
+const refreshBtn = document.querySelector('.refresh-btn');
 
 form.addEventListener('submit', (event) => {
   event.preventDefault();
 
-  const name = form.name.value;
+  const user = form.name.value;
   const score = form.score.value;
 
-  const leader = new Leaderboard(name, score);
-  UI.addLeader(leader);
-  LeaderboardStore.addLeader(leader);
+  if (!user || !score) return;
+  const newscore = {
```

Comment:
- [ ] Great work ensuring that the _**username**_ and _**score**_ can't be empty and thus can't be posted to the API endpoint through the UI. 👏